src/Migration/OldTaskFinder.php

```diff
@@ -38,7 +38,7 @@ protected function getTasks(string $path, ?string $plugin) {
 		$res = glob($path . '*Task.php');
 
 		$tasks = [];
-		if($res) {
+		if($res !== false) {
```

Comment:
I would actually much prefer the previous way..

Reply:
well then I am not sure what you mean by CS issues then ^^

Reply:
Reverted it back to how it was